db-scripts/src/main/resources/migration.sql

```diff
@@ -928,4 +928,11 @@ drop column `DRIVER_FILTER`,
 drop column `DRIVER_FILTER_ANNOTATION`,
 drop column `DRIVER_TIERS_FILTER`,
 drop column `DRIVER_TIERS_FILTER_ANNOTATION`;
-UPDATE `info` SET `DB_SCHEMA_VERSION`="2.12.7";
\ No newline at end of file
+CREATE INDEX idx_mutation_type ON mutation_event (`MUTATION_TYPE`);
+CREATE INDEX idx_cna_type ON cna_event (`ALTERATION`);
+UPDATE `info` SET `DB_SCHEMA_VERSION`="2.12.7";
```

Comment:
would be better to move these two statements to 2.12.8, the migrations should be immutable but i can change it post merge

Reply:
@inodb the merge has been performed .. we can now push the changes to the db schema into a new version. I made a previous comment in this PR about the need to also update cgds.sql with the new db schema version (as a reminder). I didn't notice that 2.12.8 was dropped in the changes.

Reply:
fyi i just fixed this here: https://github.com/cBioPortal/cbioportal/commit/6934a58d4d76f1c254046daecb9a43fd832eb4a3